### ISSUE TITLE
u8g2_font_04b_03_tr 

### ISSUE THREAD

**Issue by peppeve** [olikraus/u8g2#2524]
About this set of characters the number "1" don't have same dimension of others numbers characters, it occupe less pixel in horizontal, it is possible to have same dimension of others numbers?

**Comment by olikraus** [comment]
Isn't this the purpose for a none-monospaced font?
Of course I could create a monospaced version of the font...

**Comment by peppeve** [comment]
Yes i know, it is possible modify only the number one? or give me some suggestion.
